src/site/content/en/blog/stories-component/index.md

```diff
@@ -16,7 +17,17 @@ tags:
   - ui
 ---
 
-Let's explore and get inspired by this minimal way to layout an interactive Stories element. It's fairly similar to a carousel, but allows navigating a multidimensional array as opposed to an array. As if there's a carousel inside each carousel 🤯 
+In this post I want to share with you how I built a Stories component for
+the web that is mobile-focused, supports keyboard navigation, and works across
+browsers.
```

Comment:
Intro should quickly focus on what's in it for the reader. Facts like "mobile-focused", "supports keyboard nav", and "cross-browser support" should draw them in